Assets/Standard Assets/Characters/FirstPersonCharacter/Scripts/PlayerController.cs

```diff
@@ -80,7 +83,10 @@ private void Update()
 
     private void UpdateMovement()
     {
-        Vector2 targetDirection = new Vector2(Input.GetAxisRaw("Horizontal"), Input.GetAxisRaw("Vertical"));
+        mouseX = Input.GetAxisRaw("Vertical");
+        mouseY = Input.GetAxisRaw("Horizontal");
```

Comment:
Refactor off branch as mentioned above and this file will move and use inputHelper. Should need no changes to this file for your changes